containers/kubernetes/how-to/upgrade-cluster-node-image.mdx

```diff
@@ -0,0 +1,55 @@
+---
+meta:
+  title: How to upgrade your cluster node images
+  description: This page explains how to upgrade your node images.
+content:
+  h1: How to upgrade your cluster node images
+  paragraph: This page explains how to upgrade your node images.
+tags: kubernetes kubernetes-kapsule kapsule node image upgrade
+dates:
+  validation: 2024-01-18
+  posted: 2024-01-18
+categories:
+  - kubernetes
+---
+
+Kubernetes Kapsule clusters benefit from regular updates to their [node images](/containers/kubernetes/reference-content/kubernetes-node-images/). To profit of the latest features, it is important to stay up-to-date by upgrading these images. 
+This guide outlines the process for upgrading node images in your Kubernetes Kapsule cluster and provides instructions on updating node pool images without changing the Kubernetes version.
+
+<Message type="tip">
+  If you wish to upgrade the Kubernetes version of your cluster, refer to the documentation on [Upgrading a Kubernetes Cluster](/containers/kubernetes/how-to/upgrade-kubernets-version/).
+</Message>
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [created](/containers/kubernetes/how-to/create-cluster) a Kubernetes Kapsule cluster
+</Message>
+
+## Node Image Upgrade Process
+
+### Automatic Upgrade for Kubernetes 1.30 or later
+
+For Kubernetes clusters running version `>=1.30`, the new node image is applied automatically to all clusters created using this version. To upgrade the version of your node image, follow these steps:
+
+1. Click **Kubernetes** under **Containers** on the side menu. A list of your Kubernetes Kapsule clusters displays.
+2. Click the cluster name you wish to upgrade the pool. The cluster information page displays.
+3. Navigate to the cluster's configuration settings and select the option to upgrade node pools. This process will automatically apply the latest node image to your cluster.
+
+### Hybrid Roll-out Strategy for Kubernetes 1.29 or earlier
+
+For Kubernetes clusters running version `<=1.29`, a hybrid roll-out strategy is implemented. Follow these steps to upgrade node images in such clusters:
+
+* **Automatic deployment for new nodes:**
+   - Any newly created node is automatically deployed using the new image system.
+   - Nodes created through auto-scaling are also deployed using the new system.
+* **Transition for replaced nodes:**
+   - Replaced nodes transition to the new image system as part of the upgrade process.
+* **New node pool deployment:**
+   - Any new node pool is deployed using the new node image.
+* **Legacy nodes maintenance:**
+   - Legacy nodes, including upgraded nodes, maintain the older system for retro compatibility.
```

Comment:
Isn't it "backward compatibility" in native english? It smells french to me here

Reply:
```suggestion
   - Legacy nodes, including upgraded nodes, maintain the older system for backward compatibility.
```